src/components/create_cluster/index.js

```diff
@@ -428,14 +575,22 @@ CreateCluster.propTypes = {
   dispatch: PropTypes.func,
   provider: PropTypes.string,
   defaultInstanceType: PropTypes.string,
+  defaultVMSize: PropTypes.string,
+  defaultCPUCores: PropTypes.number,
+  defaultMemorySize: PropTypes.number,
+  defaultDiskSize: PropTypes.number,
 };
 
 function mapStateToProps(state) {
   var minAvailabilityZones = state.app.info.general.availability_zones.default;
   var maxAvailabilityZones = state.app.info.general.availability_zones.max;
   var selectedOrganization = state.app.selectedOrganization;
   var provider = state.app.info.general.provider;
-  var defaultInstanceType = 'm3.large';
+  var defaultInstanceType = 'm3.large'; // TODO
+  var defaultVMSize = 'Standard_A2_v2'; // TODO
+  var defaultCPUCores = 1; // TODO
+  var defaultMemorySize = 1; // TODO
+  var defaultDiskSize = 1; // TODO
```

Comment:
So what about these TODOs?

Reply:
I'll wire them up in another PR as far as is possible.